components/profile-about-tab/profile-about-tab.jsx

```diff
@@ -10,16 +10,16 @@ class ProfileAboutTab extends React.Component {
   renderLongBio() {
     if (!this.props.longBio) return null;
 
-    return this.props.longBio.split(`\n`).map(line => {
-      return (
```

Comment:
https://github.com/rexxars/react-markdown/blob/master/CHANGELOG.md#breaking (2nd point)

> On React >= 16.2, if no className prop is provided, a fragment will be used instead of a div. To always render a div, pass 'div' as the root renderer.

Reply:
Do we _need_ a div here, though? I assume `ReactMarkDown` already generates everything wrapped in a master element?

Reply:
Just did some test. It seems like every non-heading element is wrapped in `<p>` by default. I'll remove `<div>` from here.